### PR TITLE
Master

### PR DESCRIPTION
**Pull request by Xpressive**
Made the counting label timer in alert. To use it, just add «timeLeft» param to «show» func(or info/success/warning/danger). Alert will dismiss automatically.

### CODE REVIEW

File: Pod/Classes/JSSAlertView.swift [hunk 1]
```diff
@@ -198,6 +201,15 @@ open class JSSAlertView: UIViewController {
             yPos += ceil(textRect.height) + padding / 2
         }
         
+        if self.timerLabel != nil {
+            let timerString = timerLabel.text! as NSString
```
Comment (syky27) [inline Pod/Classes/JSSAlertView.swift]: Why do you use NSString and not String?

**Comment by syky27** [comment]
Can you add this timer to examples as well?

**Comment by syky27** [comment]
So I just noticed, that you created this PR from `master` branch... The actual version of pod is from `swift3` version, I will not be able to release new version of cocoapod, from master branch and I do not want to merge master to swift3.

**Comment by Xpressive** [comment]
Oh, sry, i'll fix this soon

**Comment by syky27** [comment]
@stakes Can you switch default branch to swift3? Or give me permission to do so?

**Comment by stakes** [comment]
@syky27 just made you an admin! I'm not sure why I didn't just do that to begin with. Thanks as always for all your work on this repo :)

**Comment by syky27** [comment]
@stakes  Sure thing bud.

**Pull request closed without merging**